util/git/client.go

```diff
@@ -195,6 +197,16 @@ func newAuth(repoURL string, creds Creds) (transport.AuthMethod, error) {
 		auth := &ssh2.PublicKeys{User: sshUser, Signer: signer}
 		if creds.insecure {
 			auth.HostKeyCallback = ssh.InsecureIgnoreHostKey()
+		} else {
+			// If we are running outside K8s cluster, we possibly need another path
+			// for SSH known hosts data.
+			if os.Getenv(common.EnvVarFakeInClusterConfig) == "true" {
```

Comment:
Can we avoid using `EnvVarFakeInClusterConfig` here. I think it would be better to always use `GetSSHKnownHostsDataPath` and set it in Procfile

Reply:
Makes sense. Maybe we can get rid of the symbolic link in `/etc/ssh/ssh_known_hosts -> /app/config/ssh/ssh_known_hosts` within the Docker image, too.